c7n/resources/cw.py

```diff
@@ -37,6 +45,35 @@ class resource_type(TypeInfo):
         cfn_type = config_type = 'AWS::CloudWatch::Alarm'
 
     retry = staticmethod(get_retry(('Throttled',)))
+    source_mapping = {
+        'describe': DescribeAlarm,
+        'config': ConfigSource
+    }
+
+
+@Alarm.action_registry.register('tag')
```

Comment:
I think we may also need remove-tag, mark-for-op and marked-for-op. Would it be better to just use universal_taggble?